lametro/base.py

```diff
@@ -0,0 +1,50 @@
+import logging
+import os
+
+from legistar.events import LegistarAPIEventScraper, WebCalendarMixin
+from scrapelib import Scraper
+
+try:
+    from .secrets import TOKEN
+except ImportError:
+    TOKEN = os.getenv("LEGISTAR_API_TOKEN", "")
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+class LAMetroAPIWebEventScraper(WebCalendarMixin, LegistarAPIEventScraper, Scraper):
```

Comment:
Since we're using this in the root scraper and the paired event stream, I broke it out into its own class.